services/node/node-base.js

```diff
@@ -5,13 +5,9 @@ const NPMBase = require('../npm/npm-base')
 const keywords = ['npm']
 
 module.exports = class NodeVersionBase extends NPMBase {
-  static get category() {
-    return 'platform-support'
-  }
+  static category = 'platform-support'
 
-  static get route() {
-    return this.buildRoute(`node/${this.path}`, { withTag: true })
-  }
+  // static route = this.buildRoute(`node`, { withTag: true })
```

Comment:
Please remove

Reply:
@calebcartwright all the comments are resolved.